#PLANNING CONFIGURATIONS:
group_name: "manipulator"
max_distance: 1
n_tests: 1000

# HUMAN-AWARE COST FUNCTION CONFIGURATION
base_frame: "world"
tool_frame: "flange"
object_type: "sphere"
n_object: 10

poi_names: ["a_forearm","b_mount_flange","b_forearm","flange"]
ssm_max_step_size: 0.001  #to make parallel version faster than serial version, max step size should be max 1/10 max_distance
ssm_n_threads: 4
max_cart_acc: 2.5
Tr: 0.15
min_distance: 0.2
v_h: 1.6

# NEURAL NETWORK CONFIGURATION
namespace: "ssm_nn"
batch_size: 10
